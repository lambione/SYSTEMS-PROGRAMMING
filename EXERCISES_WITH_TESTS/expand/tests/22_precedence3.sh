#!/bin/bash
exec "$1" tests/same_prefix_rules.txt tests/anima_rules.txt <<EOF
Algoritms are great. They are the essence of programming.  Except that
I misspelled the word "Algorithm". Anyway, I just love Algorithmic
thinking.  A good Algorithm is worth 1000 Allegories.  Well, that
doesn't make sense, I know, but I needed to somehow use the word Allegory.
EOF
