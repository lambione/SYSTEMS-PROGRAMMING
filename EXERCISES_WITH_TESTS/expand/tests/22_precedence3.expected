a wonderful course!ritms are great. They are the essence of programming.  Except that
ISI misspelled the word "a definite procedure". ASAnyway, ISI just love something to do with Algorithms
thinking.  ASA good a definite procedure is worth ten00 ASAllegories.  Well, that
doesn't make sense, ISI know, but ISI needed to somehow use the word a literary device.
